### ISSUE TITLE
Adding Genshin Impact into Favorites doesn't let BG auto-detect and maximize game's window

### ISSUE THREAD

**Issue by lainverse** [Codeusa/Borderless-Gaming#502]
I've tried all 3 methods of adding Genshin Impact into Favorites, but none of them worked and in all cases BG did nothing to maximize game window and remove the borders. However, this works just fine when I select game's process on the left side and press a button in the UI to manually make it borderless.

**Comment by lainverse** [comment]
Additional info:
1. Both launcher and game have same title "Genshin Impact", but different process names, so detect by at least process name supposed to work.
2. Genshin Impact process doesn't react to button press in UI until Full Application Refresh and disappears from list after second refresh.

**Comment by herrenP** [comment]
Have you tried this:

1. Add Genshin Impact to "Favorites"
2. Set Genshin Impact to the max resolution in-game
3. Press ALT + ENTER (this forces the game to a windowed mode with the max resolution)
4. Borderless-Gaming should now put the game into Fullscreen Borderless Window mode (might take a second or two before the software re-detects the window)

**Comment by lainverse** [comment]
Thought nobody will ever reply already. Thanks for the reply.

However, I think something went horribly wrong with Borderless Gaming on my PC in general and it probably have little to do with Genshin Impact even though it started when I installed it. Currently the only thing that works with it is maximizing current game's window by hotkey and even this cease to work eventually when application completely hangs (not immediately, but maybe after an hour just staying in memory?) and I have to kill it with Task Manager.

While it stays in background it can't detect any game window at all and process list never refresh on it own and it doesn't detect games currently running even if I manually refresh the list. Tested with HOB and Spiritfarer — both appear in the list and can be maximized, but when added to Favorites BG does nothing even though before installation of Genshin Impact it worked with them.

So, really no idea what's wrong here. Just in case, I'm using W10 Pro with latest updates as my OS.
